src/frontend/src/components/public/CommentsSidebar/index.tsx

```diff
@@ -0,0 +1,124 @@
+import { useContext, useEffect, useState } from 'react';
+import CommentSidebarProps from './interfaces';
+import { formatDate } from '../../../utils/date';
+import { API_ROUTES } from '../../../constant/apiRoutes';
+import { HttpRequest } from '../../../utils/http-request.util';
+import {
+  IPiaFormContext,
+  PiaFormContext,
+} from '../../../contexts/PiaFormContext';
+
+const CommentSidebar = ({ isRightOpen }: CommentSidebarProps) => {
```

Comment:
Similar to an above comment, I believe that the state for detecting multiple open sidebars / collapsible components should be handled by the parent.

Reply:
yes, remove this